Sources/Overload/OvUI/include/OvUI/Core/UIManager.h

```diff
@@ -132,6 +177,8 @@ namespace OvUI::Core
 		bool m_dockingState;
 		Modules::Canvas* m_currentCanvas = nullptr;
 		std::unordered_map<std::string, ImFont*> m_fonts;
-		std::string m_layoutSaveFilename = "imgui.ini";
+		std::filesystem::path m_layoutSaveFilename = "imgui.ini";
```

Comment:
Probably could be made a `std::string` (would solve the imgui ini save bug) since path features are never used with this one. Also I've noticed that it isn't saved, so restarting the editor results in the "current layout" being layout.ini again, and it would need to be loaded manually again.